cli/threadexecutor.cpp

```diff
@@ -106,7 +108,7 @@ class ThreadData
         if (mItNextFileSettings != mFileSettings.end()) {
             file = nullptr;
             fs = &(*mItNextFileSettings);
-            fileSize = 0;
```

Comment:
To make sure we always fetch the filesize we probably should initialize/set it to -1 everywhere and valid that later on. Can be done at a later date though.